fontc/src/lib.rs

```diff
@@ -1047,7 +1047,11 @@ mod tests {
         assert_eq!(1, glyph.contours().len());
         assert_eq!(
             4_usize,
-            glyph.contours().iter().map(|c| c.iter().count()).sum()
+            glyph
+                .contours()
+                .iter()
+                .map(|c| c.iter().count())
+                .sum::<usize>()
```

Comment:
weird that this had to change?

Reply:
Yeah, I'm fuzzy why but clippy was quite insistent